readme.md

```diff
@@ -497,7 +497,7 @@ Make sure to see [contributing.md](/contributing.md) for instructions on contrib
 * [Kitten](k/Kitten.ktn)
 * [Kivy](k/Kivy.py)
 * [Koa](k/Koa.js)
-* [Kobeři-C](k/Kobe%C5%99i-C.koberice)
+* [Kobeři C](k/Kobe%C5%99i%20C.koberice)
```

Comment:
The renamings should be rechecked. For example, [Kobeři-C](https://github.com/fpeterek/Koberi-C) contains the dash in the name.

Reply:
Thanks for the reminder, edited